common/stats/src/main/java/org/thingsboard/server/common/stats/DefaultTbPrintStatsExecutorService.java

```diff
@@ -25,23 +25,24 @@
 import java.util.concurrent.TimeUnit;
 
 @Component
-public class ScheduledLogExecutorComponent {
+public class DefaultTbPrintStatsExecutorService implements TbPrintStatsExecutorService {
 
-    private ScheduledExecutorService schedulerLogExecutor;
+    private ScheduledExecutorService service;
 
     @PostConstruct
     public void init() {
-        schedulerLogExecutor = Executors.newSingleThreadScheduledExecutor(ThingsBoardThreadFactory.forName("sql-log"));
+        this.service = Executors.newSingleThreadScheduledExecutor(ThingsBoardThreadFactory.forName("tb-print-stats-scheduler"));
     }
 
     @PreDestroy
-    public void stop() {
-        if (schedulerLogExecutor != null) {
-            schedulerLogExecutor.shutdownNow();
+    public void destroy() {
+        if (this.service != null) {
+            this.service.shutdown();
         }
     }
 
     public void scheduleAtFixedRate(Runnable command, long initialDelay, long period, TimeUnit unit) {
```

Comment:
Method called scheduleAtFixedRate but it call service.scheduleWithFixedDelay(). Please rename it to:

```suggestion
    public void scheduleWithFixedDelay(Runnable command, long initialDelay, long period, TimeUnit unit) {
```

In addition, please add new method to TbPrintStatsExecutorService:

```
void scheduleAtFixedRate(Runnable command, long initialDelay, long period, TimeUnit unit);
```

that will call:

```
service.scheduleAtFixedRate(command, initialDelay, period, unit);
```

Use this method in places where `ScheduledLogExecutorComponent` were used before.